misc/Kconfig

```diff
@@ -273,6 +273,24 @@ config BOOTLOADER_SRAM_SIZE
 	  This option specifies the amount of SRAM (measure in kB) reserved for
 	  when Zephyr is to act as a bootloader.
 
+config IS_MCUBOOT
+	bool
+	prompt "Act as an MCUboot bootloader"
+	default n
+	depends on XIP
+	depends on ARM
+	select IS_BOOTLOADER
+	help
+	  This option indicates that Zephyr is being built as the MCUboot
+          open source bootloader.
+ 
+config BOOTLOADER_MCUBOOT
```

Comment:
Fantastic config! This option enables when application need to chainload by MCUboot, right ?

Reply:
That's right.